content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2877,6 +2877,9 @@ Add `@ignore` to a field that you want to exclude from Prisma Client (for exampl
 
 #### Remarks
 
+- Ignored fields are excluded from the generated Prisma Client.
+- Required fields with no default value marked with `@ignore` removes the `create` and `createMany` methods in the Client API.
```

Comment:
is this actually correct @AndrewSouthpaw? I've just tested this and I was still able to execute `create` without a type error but then actually got a runtime error:

1. create schema:
```prisma
model Post {
  id        Int     @id @default(autoincrement())
  title     String 
  published Boolean @default(false)
}
```
2. `npx prisma migrate dev --name init`
3. create record:
```ts
const r = await prisma.post.create({
  data: {
    title: "Hello World"
  }
})
```
4. update schema:
```prisma
model Post {
  id        Int     @id @default(autoincrement())
  title     String  @ignore
  published Boolean @default(false)
}
```
5. `npx prisma migrate dev --name add-ignore`
6. type error in query, so remove `title` from `data` object:
```ts
const r = await prisma.post.create({
  data: {  }
})
```
7. `npm run dev`
8. violated null constraint
```
PrismaClientKnownRequestError: 
Invalid `prisma.post.create()` invocation in
/Users/nikolasburk/Desktop/ignore/script.ts:8:31

  5 // A `main` function so that we can use async/await
  6 async function main() {
  7  
→ 8   const r = await prisma.post.create(
Null constraint violation on the fields: (`title`)
```